### PR TITLE
added some more puns

### PR DESCRIPTION
**Pull request by etcadinfinitum**
Related to #4

### CODE REVIEW

**Comment by etcadinfinitum** [comment]
There are 8 puns in the file now, and a blank line at the end; hopefully should be sufficient for demonstration purposes.

File: src/punlist.txt [hunk 1]
```diff
@@ -1,2 +1,9 @@
...
-reeeeee
+In version control, git commits you.
+I typically avoid conversations with robots. They just drone on.
+"beefstew" is not a good password, but "beefStewWithMushroomsAndGravy" might be stroganoff.
+A SQL statement walks into a bar and asks 2 tables if he can join them.
+A computer gets drunk by taking screenshots.
+There are 10 types of people in the world; those who understand binary and those who don't.
+"Computer science" isn't really about computers. And it isn't really a science. But here we are anyways.
+The worst thing you can do at a SQL dinner party is clear the table.
+
```
Comment (Chris-Johnston) [inline src/punlist.txt]: I think the blank line at the end might be a problem, actually.

File: src/punlist.txt [hunk 2]
```diff
@@ -1,2 +1,9 @@
-punnnnnnny
-reeeeee
+In version control, git commits you.
+I typically avoid conversations with robots. They just drone on.
+"beefstew" is not a good password, but "beefStewWithMushroomsAndGravy" might be stroganoff.
+A SQL statement walks into a bar and asks 2 tables if he can join them.
+A computer gets drunk by taking screenshots.
+There are 10 types of people in the world; those who understand binary and those who don't.
```
Comment (Chris-Johnston) [inline src/punlist.txt:6]:
```suggestion
There are 10 types of people in the world; those who understand binary and those who don't.
There are two types of people in the world; those who can extrapolate from incomplete data.
```
related

**Review by Chris-Johnston — CHANGES_REQUESTED** [review]

File: src/punlist.txt (hunk 1 above)
Comment (etcadinfinitum) [reply src/punlist.txt]: fixed in 8883e2602797976dfdfdab9cdcc4c9dd8cc29e60

**Comment by etcadinfinitum** [comment]
gonna merge this since the content is pretty low-impact

**Pull request merged**